signer/src/testing/context.rs

```diff
@@ -162,10 +162,10 @@ impl<Storage, Bitcoin, Stacks>
 
 impl<Storage, Bitcoin, Stacks, Emily> Context for TestContext<Storage, Bitcoin, Stacks, Emily>
 where
-    Storage: DbRead + DbWrite + Clone + Sync + Send,
-    Bitcoin: BitcoinInteract + Clone + Send + Sync,
-    Stacks: StacksInteract + Clone + Send + Sync,
-    Emily: EmilyInteract + Clone + Send + Sync,
+    Storage: DbRead + DbWrite + Clone + Sync + Send + 'static,
```

Comment:
What was it that made these all need to be 'static?

Reply:
I was getting some compilation error without the `'static` bound on the Stacks client. I'm not sure if I got the error with all of them or not (certainly not with Emily). I did not revisit whether they could be removed after the code was finished though, so it's possible that we can remove the `'static` bounds now.